spec/resources/jobs_spec.rb

```diff
@@ -28,5 +28,12 @@ def find_or_publish_workbook(name)
       jobs = client.jobs.list
       expect(jobs.to_a.last.keys).to include('createdAt', 'id', 'jobType', 'priority', 'status')
     end
+
+    it 'can filter jobs in a site' do
+      workbook = find_or_publish_workbook('test')
+      client.workbooks.refresh(workbook_id: workbook['id'])
+      jobs = client.jobs.list(query: 'filter=jobType:eq:refresh_extracts,status:eq:Success')
```

Comment:
would it be a cleaner interface if we could do
```
client.jobs.list(query: { filter: 'jobType:eq:refresh_extracts,status:eq:Success' })
```
?

Reply:
I would like to accept a hash instead, but I feel like the workarounds for the colon-encoding issue would make this not so clean. We could still support hashes later, but for now this only supports strings.